hive/src/test/java/com/netease/arctic/hive/TestHMS.java

```diff
@@ -71,12 +71,16 @@ public HiveMetaStoreClient getHiveClient() {
     return mockHms.getClient();
   }
 
+  public int getMetastorePort() {
+    return mockHms.getMetastorePort();
+  }
+
   public String getWareHouseLocation() {
     return mockHms.getWareHouseLocation();
   }
 
   @Override
-  protected void before() throws Throwable {
+  public void before() throws Exception {
```

Comment:
Why need we change the method to public?

Reply:
In spark module, the SparkTestContext has to call this method because I had change the version of junit from 4 to 5, the the resource annotation is discarded.